MANIFEST.in

```diff
@@ -1,3 +1,3 @@
-include Makefile  *.txt *.md
```

Comment:
The make file is unusable by clients and doesn't seem to appear the binary distribution anyway